src/components/Input/inputRules.ts

```diff
@@ -1,11 +1,17 @@
 const inputRules = {
   root: ({ props, variables }) => {
+    const { fluid } = props
+
     return {
       display: 'inline-flex',
       position: 'relative',
       alignItems: 'center',
       justifyContent: 'flex-end',
       outline: 0,
+      ...(fluid && {
+        width: '100%',
+        maxWidth: '100%',
```

Comment:
why we need to set `maxWidth` in addition?

Reply:
removing it; not needed